### DISCUSSION TITLE
suggestion: sort plugins alphabetically

### DISCUSSION THREAD

**Discussion by Ryder17z** [OpenRCT2/OpenRCT2#17222] [Feature requests]
A simple UI improvement.
[unsorted example](https://user-images.githubusercontent.com/2000703/168597843-b3d5c7b8-a05f-4917-8c77-12fc667d747a.png)

**Comment by pfroud** [comment]
Where in the code to make the change:

https://github.com/OpenRCT2/OpenRCT2/blob/d8ba574e6cc81a96b93ebd53b0e9423ec4a72811/src/openrct2/scripting/ScriptEngine.cpp#L581-L583

On second thought, we want to sort them by the `name` passed to `registerPlugin()`, not by the filename

**Comment by ZehMatt** [comment]
~~Can't you sort them in js?~~ Nevermind, I thought about something else.
